api/views/message_views.py

```
@@ -107,4 +149,70 @@ def post(self, request):
             else:
                 return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
         else:
-            return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
\ No newline at end of file
+            return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+        
+def prepare_document_data(chatbot_data):
+        """
```

Pylint (3.3.7)

[warning] Bad indentation. Found 8 spaces, expected 4

(W0311)